docs/1-introduction/1-1-getting-started/1-1-2-hello-d3d11.md

```diff
@@ -185,15 +185,15 @@ The first part calls the parent class, where `GLFW` is initialized and setup.
 ```cpp
 #define IID_PPV_ARGS(ppType) __uuidof(**(ppType)), IID_PPV_ARGS_Helper(ppType)
 ```
-which means that typing `IID_PPV_ARGS(&_dxgiFactory)` it is expanded by the compiler into `__uuidof(**(&_dxgiFactory)), IID_PPV_ARGS_Helper(_dxgiFactory)`. This funtionally means that for functions that have a parameter setup as `REFIID` and immediatly after a `[out] void**` parameter, this macro will expand the `IID_PPV_ARGS(ppType)` expression into these parameters for ease of the programmer — this can be seen with the used `CreateDXGIFactory2` method where the second last and last parameter are a `REFIID` and `void**`:
+which means that typing `IID_PPV_ARGS(&_dxgiFactory)` it is expanded by the compiler into `__uuidof(**(&_dxgiFactory)), IID_PPV_ARGS_Helper(_dxgiFactory)`. This funtionally means that for functions that have a parameter setup as `REFIID` and immediately after a `[out] void**` parameter, this macro will expand the `IID_PPV_ARGS(ppType)` expression into these parameters for ease of the programmer — this can be seen with the used `CreateDXGIFactory2` method where the second last and last parameter are a `REFIID` and `void**`:
 ```cpp
 HRESULT CreateDXGIFactory2(
         UINT   Flags,
         REFIID riid,
   [out] void   **ppFactory
 );
 ```
-`REFIID` is a typedef that is a Reference (REF) to an Interface Identifier (`IID`) `typedef IID* REFIID`. This means that `REFIID` accepts a pointer to a Interface Identifier. An Interface Identifier is a type that holds data to describe a `GUID` (Global Unique Identifier) — which is described by a `UUID` through a specific formatted string repisenting the memory of a `GUID` type ("XXXXXXXX-XXXX-XXXX-XXXXXXXX"): 
+`REFIID` is a typedef that is a Reference (REF) to an Interface Identifier (`IID`), hence the definition of `typedef IID* REFIID`. This means that `REFIID` accepts a pointer to a Interface Identifier. An Interface Identifier is a type that holds data to describe a `GUID` (Global Unique Identifier) — which is described by a `UUID` through a specific formatted string repisenting the memory of a `GUID` type ("XXXXXXXX-XXXX-XXXX-XXXXXXXX"): 
```

Comment:
What about a simple one liner

`REFIID is just a type representing a [COM](link-to-com-subsystem-in-msdn) object`

No need to elaborate what refiid guid and all that mumbojumbo is, also what UUID/GUID is should be clear.

`repisenting` is no word :P